Makefile

```
@@ -257,7 +257,7 @@ install-git-pre-push-hook:
 
 .PHONY: build-cross
 build-cross: cmd/cli/chart.tgz
-	GO111MODULE=on CGO_ENABLED=0 $(GOX) -ldflags $(LDFLAGS) -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' ./cmd/cli
+	GO111MODULE=on CGO_ENABLED=0 $(GOX) -ldflags $(LDFLAGS) -parallel=5 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' ./cmd/cli
```

I'd be in favor of removing this flag since it defaults to the number of CPUs. I don't think there's a reason this has to (or even should) match the number of binaries being built.
```suggestion
	GO111MODULE=on CGO_ENABLED=0 $(GOX) -ldflags $(LDFLAGS) -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' ./cmd/cli
```

---

can change as required? Looks like was previously building 3 binaries in parallel so upped to 5 since now 5 binaries. Does seem to run faster with this argument.